environment/src/test/java/com/sequenceiq/environment/environment/encryption/EnvironmentEncryptionServiceTest.java

```diff
@@ -133,4 +143,75 @@ void testShouldThrowExceptionWhenCloudPlatformNotAzure() {
                 () -> underTest.getEncryptionResources("NotAzure"));
         assertEquals("No Encryption resources component found for cloud platform: NotAzure", exception.getMessage());
     }
+
+    @Test
+    void testCreateEncryptionResourcesDeletionRequestShouldReturnWithANewDeletionRequest() {
+        EnvironmentDto environmentDto = EnvironmentDto.builder()
+                .withResourceCrn("crn:cdp:environments:us-west-1:dummyUser:environment:randomGeneratedResource")
+                .withId(1L)
+                .withName("envName")
+                .withCloudPlatform("AZURE")
+                .withCredential(new Credential())
+                .withLocationDto(LocationDto.builder()
+                        .withName("dummyRegion")
+                        .build())
+                .withParameters(ParametersDto.builder()
+                        .withAzureParameters(AzureParametersDto.builder()
+                                .withEncryptionParameters(AzureResourceEncryptionParametersDto.builder()
+                                        .withDiskEncryptionSetId("dummyDesId").build())
+                                .build())
+                        .build())
+                .withCreator("dummyUser")
+                .withAccountId("dummyAccountId")
+                .build();
+        CloudResource desCloudResource = CloudResource.builder()
+                .name("Des")
+                .type(ResourceType.AZURE_DISK_ENCRYPTION_SET)
+                .reference("dummyDesId")
+                .status(CommonStatus.CREATED)
+                .build();
+        when(resourceRetriever.findByResourceReferenceAndStatusAndType(any(), any(), any())).thenReturn(Optional.ofNullable(desCloudResource));
+        DiskEncryptionSetDeletionRequest deletionRequest = underTest.createEncryptionResourcesDeletionRequest(environmentDto);
+        Optional<CloudResource> dummyResource = deletionRequest.getCloudResource().stream()
+                .filter(r -> r.getType() == ResourceType.AZURE_DISK_ENCRYPTION_SET)
+                .findFirst();
+        assertNotNull(dummyResource);
+        assertEquals(dummyResource.get().getReference(), "dummyDesId");
+    }
+
+    @Test
+    void testDeleteEncryptionResourcesShouldCallDeleteDiskEncryptionSetWhenCloudPlatformAzure() {
+        when(cloudPlatformConnectors.get(any(CloudPlatformVariant.class))).thenReturn(cloudConnector);
+        when(cloudConnector.encryptionResources()).thenReturn(encryptionResources);
+        Environment env = new Environment();
+        env.setCloudPlatform("AZURE");
```

Comment:
Not needed. Please also remove from `testCreateEncryptionResourcesShouldCallCreateDiskEncryptionSetWhenCloudPlatformAzure()`.